src/main/java/remi/model/Ui.java

```diff
@@ -0,0 +1,57 @@
+package remi.model;
+
+import remi.io.Inputter;
+import remi.io.Message;
+import remi.io.Outputter;
+import remi.model.commands.CommandList;
+import remi.parsing.Parser;
+import remi.storage.Storage;
+import remi.utils.RemiError;
+
+/**
+ * Handles the input output loop, receiving commands and sending output to the console. Supposed to be the "main" class
+ * of the application.
+ */
+public class Ui {
+
+    private boolean exitLoop;
```

Comment:
boolean variables should be named to sound like boolean. (see [coding standard](https://se-education.org/guides/conventions/java/intermediate.html#naming))